libraries/Microsoft.Bot.Builder.AI.QnA/Microsoft.Bot.Builder.AI.QnA.csproj

```diff
@@ -12,6 +12,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Classes for using QnA Maker Cognitive Service on the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using Microsoft Cognitive services.</Summary>
+    <ContentTargetFolders>content</ContentTargetFolders>
   </PropertyGroup>
```

Comment:
For .schema?

Reply:
yeah, this causes content files to be included in package but not mapped into project folder